console-subscriber/src/builder.rs

```diff
@@ -149,6 +159,279 @@ impl Builder {
 
         self
     }
+
+    /// Initializes the console [tracing `Subscriber`][sub] and starts the console
+    /// subscriber [`Server`] on its own background thread.
+    ///
+    /// This function represents the easiest way to get started using
+    /// tokio-console.
+    ///
+    /// In addition to the [`TasksLayer`], which collects instrumentation data
+    /// consumed by the console, the default [`Subscriber`][sub] initialized by this
+    /// function also includes a [`tracing_subscriber::fmt`] layer, which logs
+    /// tracing spans and events to stdout. Which spans and events are logged will
+    /// be determined by the `RUST_LOG` environment variable.
+    ///
+    /// **Note**: this function sets the [default `tracing` subscriber][default]
+    /// for your application. If you need to add additional layers to this
+    /// subscriber, see [`build`].
+    ///
+    /// [default]: https://docs.rs/tracing/latest/tracing/dispatcher/index.html#setting-the-default-subscriber
+    /// [sub]: https://docs.rs/tracing/latest/tracing/trait.Subscriber.html
+    /// [`tracing_subscriber::fmt`]: https://docs.rs/tracing-subscriber/latest/tracing-subscriber/fmt/index.html
+    /// [`Server`]: crate::Server
+    ///
+    /// ## Configuration
+    ///
+    /// Tokio console subscriber is configured with sensible defaults for most
+    /// use cases. If you need to tune these parameters, several environmental
+    /// configuration variables are available:
+    ///
+    /// | **Environment Variable**            | **Purpose**                                                               | **Default Value** |
+    /// |-------------------------------------|---------------------------------------------------------------------------|-------------------|
+    /// | `TOKIO_CONSOLE_RETENTION_SECS`      | The number of seconds to accumulate completed tracing data                | 3600s (1h)        |
```

Comment:
I know this isn't your mistake (since you presumably just cut-and-pasted the table from the `init` docs), but there are typos in the environment variable names from the `init` docs.

See this commit for how I addressed it in a parallel PR: https://github.com/tokio-rs/console/pull/198/commits/aad08331a8bb90c03ba9b2c3d0c893413f65151d